ui/packages/catalog-ui-search/src/main/webapp/component/builder/builder.view.js

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+
+
+define([
```

Comment:
Since this is a new component we should use the new style of module rather than the old.  

```
const Backbone = require('backbone');
const Marionette = require('marionette');
const _ = require('underscore');
const $ = require('jquery');
const template = require('./builder.hbs');
const CustomElements = require('js/CustomElements');
const PropertyCollectionView = require('component/property/property.collection.view');
const LoadingView = require('component/loading/loading.view');
const AvailableTypesView = require('component/available-types/available-types.view');
const DropdownView = require('component/dropdown/dropdown.view');
const metacardDefinitions = require('component/singletons/metacard-definitions');
const PropertyView = require('component/property/property.view');
const Property = require('component/property/property');
```

And then instead of returning do a `module.exports = whatYouWereReturning`.

Reply:
done